tools/CleanMSSQLLocalDB.cmd

```diff
@@ -0,0 +1,6 @@
+echo %PATH%
```

Comment:
Not sure I've seen the current error before. Would restoring `@echo off` get the script working again?
```suggestion
@echo off
```

Reply:
I am not sure how `echo` would produce that error, perhaps %PATH% contains newline characters

Reply:
I don't know either. I'll see if there's a way to make sure prepended paths are syntactically correct later tonight. The dog is getting insistent about her walk&hellip;

Separately, should we retarget this to 'release/6.4'?

Reply:
Yeah, it wasn't coming from that line. Anyway seems like it's working 🤷‍♂

Reply:
(Have to admit it's been fun running up the number of commits in this "simple" PR 😈)

Reply:
Now you understand why we were hesitant to do this at all